### DISCUSSION TITLE
Volumetric heat source in FDS

### DISCUSSION THREAD

**Discussion by uniqueasad** [firemodels/fds#12831] [General]
Hi,

I want to give volume with respect of Time. I have used  INTERNAL_HEAT_SOURCE = 0.,300. / method to define constant heat generation rate. 

Is there any way to define transient volumetric heat source in the solid?

**Comment by drjfloyd** [comment]
There is currently (FDS 6.9.1) no RAMP input for INTERNAL_HEAT_SOURCE.

What is it you are trying to accomplish with INTERNAL_HEAT_SOURCE?
